Resources/Storage/SRM2Storage.py

```diff
@@ -82,7 +82,7 @@ def __init__( self, storageName, protocol, path, host, port, spaceToken, wspath
     self.gfalRetry = gConfig.getValue( "/Resources/StorageElements/GFAL_Retry", 3 )
 
     # # set checksum type, by default this is 0 (GFAL_CKSM_NONE)
-    self.checksumType = gConfig.getValue( "/Resources/StorageElements/ChecksumType", 0 )
+    self.checksumType = gConfig.getValue( "/Resources/StorageElements/ChecksumType", None )
```

Comment:
exactly. This getValue is really dangerous beast.... and the signature is very misleading :
  def getValue( self, optionPath, defaultValue = None ):

defaultValue is all, but a default value